src/ngraph_capture_variables.cc

```diff
@@ -28,25 +28,32 @@ namespace tensorflow {
 namespace ngraph_bridge {
 
 //
-// Utility function to check if placement on the NGRAPH device has been
+// Utility function to check if placement on the NGRAPH has been
 // requested.
-//
-// FIXME(amprocte): stubbed out for now because NGRAPH device is gone.
-//
-static bool NGraphPlacementRequested(const Node* node) { return true; }
+static bool NGraphPlacementRequested(const Node* node,
+                                     std::vector<string> skip_these_nodes) {
+  for (string& f : skip_these_nodes) {
```

Comment:
You can avoid the for loop if you use a set.

Reply:
Currently, I used a vector because it was small list. Will be replacing it with a map since we will need the name as well as the id. This is being done in the separate PR so did not want make a lot of changes here.